who_l3_smart_tools/core/terminology/who/schema.py

```diff
@@ -0,0 +1,40 @@
+import slugify
+from who_l3_smart_tools.core.terminology.schema import ConceptSchema
+
+def dash_preserving_slugify(text):
```

Comment:
This probably belongs in some kind of utility class right?

Reply:
It does ideally but I prefer to refactor when needed. If it is only used here then it should stay here until there is a need to share it.

Reply:
Sorry, I phrased that badly. What I meant was we already have the `who_l3_smart_tools.utils` namespace in this package, and it seems like a more logical place for this function to go.